src/main/java/dog/pawbook/model/managedentity/owner/Owner.java

```diff
@@ -99,6 +102,14 @@ public String toString() {
         return builder.toString();
     }
 
+    public void addDogId(int dogId) {
+        this.dogIdList.add(dogId);
+    }
```

Comment:
as mentioned in the AddDogCommand portion, it will not necessary to directly modify the Entity objects since all attributes are constant throughout the lifetime of the object. This is to align to the original AB3 codebase whereby the edit command also modifies by creating an updated instance of `Person` before asking the model to set it as the new one.